from-repo/c-cpp/README.md

```diff
@@ -0,0 +1,14 @@
+# Auto-build oss-fuzz projects from a GitHub repo URL
```

Comment:
Can we put this all under `experimental`? 

We can put all the other language support there too. e.g. `experimental/java`

Reply:
yes, done